functests/2_performance_update/updating_profile.go

```diff
@@ -55,6 +56,70 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	Context("Verify GloballyDisableIrqLoadBalancing Spec field", func() {
+		It("[test_id:36150] Verify that IRQ load balancing is enabled/disabled correctly", func() {
```

Comment:
just curious, is it possible to make this test as part of common "performance profile updating" test?
https://github.com/openshift-kni/performance-addon-operators/blob/dfe203e14f050dda0f8f39dda24b567db6b91782/functests/2_performance_update/updating_profile.go#L123

It may help us to reduce wasting time on "updating / reverting back" node tunnings.

Reply:
I thought about it, but is a very fast test, is only updating the tuned profile, all the test takes a few seconds.
Also the test has checks before and after the update, it doesn't feel right to check the business logic in "Before All"